From 77bae7e7ea5c02f5be0d59dabeae6c3c09571fbe Mon Sep 17 00:00:00 2001
From: Elliott Sales de Andrade <quantum.analyst@gmail.com>
Date: Wed, 24 Jan 2018 02:51:55 -0500
Subject: [PATCH] Add libdl on Unix-like systems.

Not linking with libdl causes errors when -z defs is in the linker
flags.

Signed-off-by: Elliott Sales de Andrade <quantum.analyst@gmail.com>
---
 setupext.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/setupext.py b/setupext.py
index d0f3f2070..6d2c3e296 100644
--- a/setupext.py
+++ b/setupext.py
@@ -1493,6 +1493,8 @@ class BackendTkAgg(OptionalBackendPackage):
         if sys.platform == 'win32':
             # PSAPI library needed for finding Tcl / Tk at run time
             ext.libraries.extend(['psapi'])
+        elif sys.platform != 'darwin':
+            ext.libraries.extend(['dl'])
 
 
 class BackendGtk(OptionalBackendPackage):
-- 
2.14.3

